test/unit/effects.js

```diff
@@ -1527,7 +1527,7 @@ QUnit.test( "Animate callbacks have correct context", function( assert ) {
 	this.clock.tick( 10 );
 } );
 
-QUnit.test( "User supplied callback called after show when fx off (#8892)", function( assert ) {
+QUnit[ jQuery.find.compile ? "test" : "skip" ]( "User supplied callback called after show when fx off (#8892)", function( assert ) {
```

Comment:
Can we keep this test? The right way to do it would be testing visibility with `supportjQuery`, but it would also be acceptable to replace use of `:hidden`/`:visible`.